src/components/content-tab-previews.tsx

```diff
@@ -0,0 +1,241 @@
+import offlineIcon from './offline-icon';
+import { ScreenshotDemoSite } from './screenshot-demo-site';
+import { Tooltip } from './tooltip';
+
+interface ContentTabPreviewsProps {
+	selectedSite: SiteDetails;
+}
+
+function EmptyGeneric( {
+	children,
+	selectedSite,
+}: PropsWithChildren< { selectedSite: SiteDetails } > ) {
+	const { __ } = useI18n();
+	return (
+		<div className="p-8 flex justify-between max-w-3xl gap-4">
+			<div className="flex flex-col">
+				<div className="a8c-subtitle mb-1">{ __( 'Share a preview of your Studio site' ) }</div>
+				<div className="w-[40ch] text-a8c-gray-70 a8c-body">
+					{ __(
+						'Get feedback from anyone, anywhere with a free hosted preview of your Studio site.'
+					) }
+				</div>
+				<div className="mt-6">
+					{ [
+						__( 'Create up to 10 preview links for free.' ),
+						__( 'Preview links expire 7 days after the last update.' ),
+						createInterpolateElement( __( 'Powered by <a> WordPress.com</a>.' ), {
+							a: (
+								<Button
+									variant="link"
+									className="whitespace-pre"
+									onClick={ () =>
+										getIpcApi().openURL(
+											'https://wordpress.com/?utm_source=studio&utm_medium=referral&utm_campaign=demo_sites_onboarding'
+										)
+									}
+								/>
+							),
+						} ),
+					].map( ( text ) => (
+						<div
+							key={ typeof text === 'string' ? text : 'wordpress-com' }
+							className="text-a8c-gray-70 a8c-body flex items-center"
+						>
+							<Icon className="fill-a8c-blueberry ltr:mr-2 rtl:ml-2 shrink-0" icon={ check } />
+							{ text }
+						</div>
+					) ) }
+				</div>
+				{ children }
+			</div>
+			<div className="flex flex-col shrink-0 items-end">
+				<ScreenshotDemoSite site={ selectedSite } />
+			</div>
+		</div>
+	);
+}
+
+function NoAuth( { selectedSite }: React.ComponentProps< typeof EmptyGeneric > ) {
+	const isOffline = useOffline();
+	const { __ } = useI18n();
+	const { authenticate } = useAuth();
+	const offlineMessage = __( "You're currently offline." );
+
+	return (
+		<EmptyGeneric selectedSite={ selectedSite }>
+			<div className="mt-8">
+				<Tooltip disabled={ ! isOffline } icon={ offlineIcon } text={ offlineMessage }>
+					<Button
+						aria-description={ isOffline ? offlineMessage : '' }
+						aria-disabled={ isOffline }
+						variant="primary"
+						onClick={ () => {
+							if ( isOffline ) {
+								return;
+							}
+							authenticate();
+						} }
+					>
+						{ __( 'Log in to WordPress.com' ) }
+						<Icon className="ltr:ml-1 rtl:mr-1 rtl:scale-x-[-1]" icon={ external } size={ 21 } />
+					</Button>
+				</Tooltip>
+			</div>
+			<div className="mt-3 w-[40ch] text-a8c-gray-70 a8c-body">
+				<Tooltip
+					disabled={ ! isOffline }
+					icon={ offlineIcon }
+					text={ offlineMessage }
+					placement="bottom-start"
+				>
+					{ createInterpolateElement(
+						__(
+							'A WordPress.com account is required to create preview links. <a>Create a free account</a>'
```

Comment:
Let's use Preview sites instead of links.

```suggestion
							'A WordPress.com account is required to create preview sites. <a>Create a free account</a>'
```

Reply:
Updated in https://github.com/Automattic/studio/pull/846/commits/64f6aac0877b162c493700748d4f000acbb5f366